app/uk/gov/hmrc/exports/models/declaration/Locations.scala

```diff
@@ -30,12 +30,18 @@ object GoodsLocation {
   implicit val format: OFormat[GoodsLocation] = Json.format[GoodsLocation]
 }
 
-case class OfficeOfExit(officeId: Option[String], presentationOfficeId: Option[String], circumstancesCode: Option[String])
+case class OfficeOfExit(officeId: Option[String], presentationOfficeId: Option[String], circumstancesCode: Option[String], answer: String)
```

Comment:
Another thought occurred to me last night.   If the "answer" is not optional then it will require that the BE and FE PRs are deployed at the same time and this might not be possible due to other conflicting tickets.   Not 100% sure that this is (always) true but it needs some consideration.

Ideally, we want to be able to release a BE change first (that is compatible with the existing FE) before we release the FE change.